taxonomies/food/ingredients.txt

```diff
@@ -4309,243 +3311,9 @@ vegetarian:en: yes
 
 ################################ MILK POWDERS ##################################
```

Comment:
This section can probably be removed and the two preparations left either be moved to the previous or the next section.